include/mbedtls/md.h

```diff
@@ -63,7 +63,7 @@ typedef enum {
     MBEDTLS_MD_RIPEMD160, /**< The RIPEMD-160 message digest. */
 } mbedtls_md_type_t;
 
-#if defined(MBEDTLS_SHA512_C)
+#if defined(MBEDTLS_SHA512_C) || defined(MBEDTLS_SHA384_C)
```

Comment:
Actually we could even go further and have SHA512 -> 64 else SHA384 -> 48 else 32. (Or even else SHA256 -> 32 else SHA224 -> 28 else 20.) (20 could be SHA-1 or RIPE MD-160)